tide_media.install

```diff
@@ -156,7 +156,12 @@ function tide_media_update_8002() {
  */
 function tide_media_update_8003() {
   $database = Drupal::database();
-
+  $line_breaks = [
+    '\'\r\n\'',
+    '\'\n\r\'',
```

Comment:
@vincent-gao `\n\r` is not a linebreak. Valid linebreaks are `\r\n` (CRLF - Windows), `\r` (CR - MacOS 9), and `\n` (LF - *nix).

Reply:
Thanks.